### PR TITLE
FIX when editing user superadmin he was goin back to admin

### PR DESCRIPTION
**Pull request by atm-quentin**
#FIX when editing a superadmin user, he was going back to admin
because entity is automatically fill

### CODE REVIEW

File: htdocs/user/card.php [hunk 1]
```diff
@@ -2033,9 +2032,10 @@
 								</script>';
 						}
 
-						$checked=(($object->admin && ! $object->entity) ? ' checked' : '');
-						print '<input type="checkbox" name="superadmin" value="1"'.$checked.' /> '.$langs->trans("SuperAdministrator");
+
 					}
+                    $checked=(($object->admin && ! $object->entity) ? ' checked' : '');
+                    print '<input type="checkbox" name="superadmin" value="1"'.$checked.' /> '.$langs->trans("SuperAdministrator");
```
Comment (hregis) [inline htdocs/user/card.php:2038]: @atm-quentin 
it's wrong !! only a superadmin can downgrade a superadmin !!

**Comment by hregis** [comment]
@atm-quentin 
What is the problem exactly ?

**Comment by atm-quentin** [comment]
When you are a superadmin and you change town of an other superadmin, he goes back to admin

**Comment by hregis** [comment]
@atm-quentin @eldy 

this can happen when a second "superadmin" is created before enabling the centralized management of users in the main entity, once this option is activated, a "superadmin" modified by another superadmin becomes an "admin" again. The fact of having only one "superadmin" in "transversal" mode was a limitation of the time, I do not know why anymore! I will correct Dolibarr to be able to create several "superadmin" in "transversal" mode.

**Comment by hregis** [comment]
@atm-quentin @eldy 

fixed with https://github.com/Dolibarr/dolibarr/pull/10671

**Comment by atm-quentin** [comment]
Thanks

**Pull request closed without merging**